React-WhatsNewInReact18.MD

```diff
@@ -72,7 +72,7 @@ The actual updates happen at the end, and the component renders `3`.
 
 ## startTransition (opt-in)
  
-`startTransition` can be used to mark UI updates that do not need urgent resources for updating. For eg: when typing in a typeahead field, there are two things happening - a blinking cursor that shows visual feedback of your content being typed, and a search functionality in the background that searches for the data that is typed. 
+`startTransition` can be used to mark UI updates that do not need urgent resources for updating. For example: when typing in a typeahead field, there are two things happening - a blinking cursor that shows visual feedback of your content being typed, and a search functionality in the background that searches for the data that is typed. 
```

Comment:
`e.g.` already stands for "for example": https://www.merriam-webster.com/dictionary/e.g.